docs/jwst/extract_2d/main.rst

```diff
@@ -32,9 +32,9 @@ NIRCAM GRISM and NIRISS WFSS observations.
 
 For NIRSPEC:
 
-If the step parameter ``which_subarray`` is left unspecified, the default behavior is
+If the step parameter ``slit_name`` is left unspecified, the default behavior is
 to extract all slits which fall within a detector. Only one slit may be extracted by
```

Comment:
Maybe instead of saying "all slits which fall within a detector" it would be more correct - for all situations involving full-frame and subarray exposures - to say something like "all slits that fall within the limits of the image." And how much of the slit/spectrum trace has to fall within the image to be included in the list of slits to process? Does the entire trace (i.e. covering all wavelengths) have to be inside the image, or is there some minimum portion that must fall inside the image, or what? I'm thinking in particular about the SUB512 and SUB1024 subarrays that can be used with the S1600A1 slit. Those subarrays will likely truncate the trace (at one end or the other, if not both), while leaving some part of it available in the image.

Reply:
If any part of the trace projects on the detector it will be extracted as a separate slit.

But it's not sufficient for a slit to be in the image to be extracted. It has to be projected within the detector limits as well. Part of the model is run on every open slit to determine if it projects on the detector or not. This is especially true for the high-resolution gratings and NRS2. Or am I  mixing terminology?